java-lib/src/main/java/com/wavefront/common/TraceConstants.java

```diff
@@ -0,0 +1,13 @@
+package com.wavefront.common;
+
+/**
+ * Trace constants.
+ *
+ * @author Anil Kodali (akodali@vmware.com)
+ */
+public abstract class TraceConstants {
+  // Span References
+  // See https://opentracing.io/specification/ for more information about span references.
+  public static final String FOLLOWS_FROM_KEY = "followsFrom";
```

Comment:
Okay for now. But ideally this should go in `wavefront-sdk-java`.

Reply:
Can add a TODO for this so that we can track this.